vendor.mod

```diff
@@ -168,3 +179,6 @@ require (
 	google.golang.org/protobuf v1.28.1 // indirect
 	k8s.io/klog/v2 v2.80.1 // indirect
 )
+
+// FIXME(thaJeztah): remove this replace once we can; see https://github.com/moby/buildkit/pull/3410 and https://github.com/spdx/tools-golang/pull/174
+replace github.com/spdx/tools-golang => github.com/jedevc/spdx-tools-golang v0.0.0-20221205121515-6fe9d226281a
```

Comment:
Looks like we need a replace rule, pending https://github.com/spdx/tools-golang/pull/174 (see https://github.com/moby/buildkit/pull/3410)